### ISSUE TITLE
I have been trying this out, but I get to the same sticking point running this locally

### ISSUE THREAD

**Issue by jaffamonkey** [epiclabs-io/docker-cucumber-protractor-chrome#1]
I have been trying this out, but I get to the same sticking point running this locally (same result when I tried to run on travisci). Tired adjusting package version for webdriver-manager, and updating with specific versions of browser drivers, without success.

`$ docker run -it --volume /home/travis/build/jaffamonkey/docker-cucumber-protractor-chrome/example:/tests epiclabs/e2e-test-runner --baseUrl http://google.com --specs ./tests/features/google.feature
webdriver-manager: using local installed version 12.0.6
webdriver-manager: using local installed version 12.0.6
[12:39:14] I/update - chromedriver: unzipping chromedriver_2.33.zip
[12:39:14] I/start - java -Djava.security.egd=file:///dev/./urandom -Dwebdriver.chrome.driver=/src/node_modules/webdriver-manager/selenium/chromedriver_2.33 -Dwebdriver.gecko.driver=/src/node_modules/webdriver-manager/selenium/geckodriver-v0.19.1 -jar /src/node_modules/webdriver-manager/selenium/selenium-server-standalone-3.7.1.jar -port 4444
[12:39:14] I/start - seleniumProcess.pid: 27
[12:39:14] I/start - Selenium Standalone has exited with code 1
[12:39:14] I/update - chromedriver: setting permissions to 0755 for /src/node_modules/webdriver-manager/selenium/chromedriver_2.33
[12:39:16] I/update - geckodriver: unzipping geckodriver-v0.19.1.tar.gz
[12:39:16] I/update - geckodriver: setting permissions to 0755 for /src/node_modules/webdriver-manager/selenium/geckodriver-v0.19.1

# Here is point where process just hangs, running on travis, i get extra output below

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated`

**Comment by ricbra** [comment]
I experienced the same problem. Found a solution yet? Or another ready to use docker image for cucumberjs, selenium and chrome?

**Comment by aescarcha** [comment]
Hi guys, thank you for trying this. The issue is probably in the npm versions, I'll try to patch this during this week.

**Comment by aescarcha** [comment]
This should be fixed, the line

    docker run -it --volume  /Users/alvaroescarcha/epic/github/docker-cucumber-protractor-chrome/example:/tests epiclabs/e2e-test-runner --baseUrl https://www.google.com --specs /tests/features/*.feature

Works for me now, I've frozen the package.json versions, also the error didn't show up because the output is redirected to `/dev/null` to avoid spam.

**Comment by robsoncassol** [comment]
the same problem here, this last comment don't helped :(

**Comment by aescarcha** [comment]
Hi Rob, could you try to launch the container with `--debugger` ? Please copy here the trace.

**Comment by robsoncassol** [comment]
`[15:36:21] I/update - geckodriver: file exists /src/node_modules/webdriver-manager/selenium/geckodriver-v0.19.1.tar.gz
[15:36:21] I/update - geckodriver: unzipping geckodriver-v0.19.1.tar.gz
[15:36:22] I/update - geckodriver: setting permissions to 0755 for /src/node_modules/webdriver-manager/selenium/geckodriver-v0.19.1
[15:36:22] I/update - geckodriver: geckodriver-v0.19.1 up to date
Waiting for selenium on 127.0.0.1
Waiting for selenium on 127.0.0.1
Waiting for selenium on 127.0.0.1
`
Selenium doesn't work

**Comment by kjellski** [comment]
I've created a PR with new versions for java and node, it fixed the issue for me, maybe you can test that too @robsoncassol :)

**Comment by kjellski** [comment]
I think I've figured the problem out, created a new PR (#3) that should solve this.

The problem was that the `webdriver update` was started in parallel to the `webdriver start`.
